VCS/1. User Guides/15. 管理資料磁碟.md

```diff
@@ -1 +1,212 @@
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_d79c9ac00e3961f1b1f23c7ed57883b5.png)
+
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b> (Not yet supported)</summary>
+
+<br>
+
+</details>
+
+<br>
+
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+
+1. If you need to change to another disk type, or need to change the disk size, see [<ins>HowTo : Change data disk</ins>](https://man.twcc.ai/@twccdocs/howto-bss-replace-data-vol-en).
+2. After the data disk is mounted the VCS instance, it needs to be initialized before using. For the steps, see: 
+- [<ins>HowTo: Initialize data disk- Linux instance</ins>](https://man.twcc.ai/@twccdocs/howto-bss-init-vol-linux-en)
+- [<ins>HowTo: Initialize data disk- Windows instance</ins>](https://man.twcc.ai/@twccdocs/howto-bss-init-vol-windows-en)
+:::
+
+## Detach from VCS instance
+
+If you need to delete the data disk or mount other VCS instance, please detach the disk from the original VCS instance before performing other operations.
+
+
+For detailed steps of uninstalling and detaching data disks, see [Detach data disk](https://man.twcc.ai/@twccdocs/vcs-vds-guide-detach-data-disk-en).
+
+
+
+## Delete data disk
+
+When the disk is not needed, you can delete the disk. Once deleted, <ins>**it is not billed anymore**</ins>.
+
+
+:::danger
+{%hackmd @docsharedstyle/important-en %}
+1. Be careful before deleting the data disk. Once the disk is deleted, it cannot be recovered.
+2. If the state of the data disk is **`IN-USE`**, the disk must be detached from the VCS instance to enter in **`AVAILABLE`** state first, then it can be deleted.
+3. When a snapshot of the data disk exists, the disk cannot be deleted. Please delete the disk snapshot before deleting the data disk.
+:::
+
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+- Enter **Data Disk Management** page > select the data disk(s) you want to delete > click **DELETE** at the top to delete the disk.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_6d9947298979e85c4062f988994e51e4.png)
+
+
+- Or click "<i class="fa fa-ellipsis-v fa-20" aria-hidden="true"></i>" next to the data disk &nbsp;> click **DELETE**. 
+- Or enter **Data Disk Details** page> click "**DELETE**" at the top.
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b></summary>
+
+<br>
+
+### Commands
+
+```bash
+$ twccli rm vds -id  #Data disk ID 
+                [-f] #Whether a force deletion
+```
+
+:::info
+{%hackmd @twccdocs/cli-parameter-note-en %}
+:::
+
+### For example
+
+- Delete data disk with ID `376749`
+```bash
+$ twccli rm vds -id 376749
+```
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_e2a0873513f1cc8f60be01a78ae3b456.png)
+
+- Forced to delete (without warning window) the data disk with ID `376716`
```

Comment:
[SUG]

Forced to delete (without warning window) > Force delete (without warning)